### DISCUSSION TITLE
Vocabulary Releases

### DISCUSSION THREAD

**Discussion by CHerzog-CAI** [OHDSI/Vocabulary-v5.0#638] [Q&A] [answered]
I can't tell due to so many ellipses what all is going to be released from this chart,
[https://github.com/OHDSI/Vocabulary-v5.0/wiki/Release-planning---quarterly-updated#roadmap-2022-q1](url).  Can someone point me to the none truncated and ellipse version of this?

What I am really searching for is, when OHDSI will be releasing HemOnc in 2022?

**Comment by mik-ohdsi** [comment, marked as answer]
Hi @CHerzog-CAI  - this is a roadmap, not necessarily reflecting the exact timeline in which things will happen. This representation is not ideal, as on export some smaller bars are unfortunately shrunk and appear with [...] . Now, this has happened to the HemOnc refresh, which was scheduled for June but as we experience some delays in the slightly more important SNOMED release and have to reassign resources, will be shifted to July/August. In the future I will refrain from putting the "Refresh:" at the beginning so that the more relevant part stays visible. Thanks for your input!

**Reply by CHerzog-CAI** [reply]
Thank you for your prompt response.  I also appreciate the offered up information and resource constraints.
